<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lrcwidget</class>
 <widget class="QWidget" name="lrcwidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lrcWidget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭歌词</string>
   </property>
  </widget>
  <widget class="QLabel" name="labCov">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="lrc_list">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>301</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>711</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: #B0E0E6; /* 轨道背景颜色，淡蓝色 */
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: #FFFFFF; /* 滑块颜色，白色 */
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}

QSlider::add-page:horizontal {
    background: #E0FFFF; /* 滑块右侧颜色，更淡的蓝色 */
}

QSlider::sub-page:horizontal {
    background: #87CEFA; /* 滑块左侧颜色，浅天蓝色 */
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labProcess">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>00:00/00:00</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>267</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/volumn.bmp</normaloff>:/images/images/volumn.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="soundSlider">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: #B0E0E6; /* 轨道背景颜色，淡蓝色 */
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: #FFFFFF; /* 滑块颜色，白色 */
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}

QSlider::add-page:horizontal {
    background: #E0FFFF; /* 滑块右侧颜色，更淡的蓝色 */
}

QSlider::sub-page:horizontal {
    background: #87CEFA; /* 滑块左侧颜色，浅天蓝色 */
}
</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinBoxSpeed">
      <property name="suffix">
       <string>倍速</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMode">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>循环</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/refresh16.png</normaloff>:/images/images/refresh16.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>550</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btnPrev">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/616.bmp</normaloff>:/images/images/616.bmp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPlay">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnNext">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/630.bmp</normaloff>:/images/images/630.bmp</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
